PoseEstimation-TFLiteSwift/VideoCapture.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  VideoCapture.swift
+//  Awesome ML
+//
+//  Created by Eugene Bokhan on 3/13/18.
+//  Copyright © 2018 Eugene Bokhan. All rights reserved.
+//
+
+import UIKit
+import AVFoundation
+import CoreVideo
+
+public protocol VideoCaptureDelegate: class {
+    func videoCapture(_ capture: VideoCapture, didCaptureVideoFrame: CVPixelBuffer?, timestamp: CMTime)
```

Comment:
Why you pass optional CVPixelBuffer?
If you get a nil, that mean VideoCapture didn't capture video frame.

Reply:
Thanks. I accepted and fixed it! 💪